.github/workflows/outboxer.yml

```
@@ -42,3 +42,30 @@ jobs:
       - name: Execute test
         working-directory: ./integration-test
         run: npm run test
+
+# Authentication to GitHub container registry is only possible with an personal access token (PAT)
+# The publish job requires a secret named `GHCR_PAT` which must contain a PAT with the `read:packages`
+# and `write:packages` scope.
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: github.ref == 'refs/heads/ghcr-test'
```

Before merging the branch name must be changed:

```suggestion
    if: github.ref == 'refs/heads/main'
```